package.json

```diff
@@ -119,6 +119,7 @@
   },
   "packageManager": "pnpm@9.6.0+sha512.38dc6fba8dba35b39340b9700112c2fe1e12f10b17134715a4aa98ccf7bb035e76fd981cf0bb384dfa98f8d6af5481c2bef2f4266a24bfa20c34eb7147ce0b5e",
   "dependencies": {
+    "browserstack-node-sdk": "1.34.17",
```

Comment:
i left this by mistake but its not a big deal i dont wanna back it out for the sake of this pr, i will fix this going forward